scss/modules/_forms.scss

```diff
@@ -58,57 +90,6 @@
     }
   }
 
-  input:focus,
-  textarea:focus {
-    border-color: $brand-color;
```

Comment:
It is now difficult to see which field is in focus. I think we need something like this.